### DISCUSSION TITLE
Audacious 4.3.1 on Ubuntu 22.04 - Only one Output Pulseaudio

### DISCUSSION THREAD

**Discussion by VincentLeclercq** [audacious-media-player/audacious#1402] [General]
Hi everyone,

I simply installed Audacious (4.3.1) on Ubuntu 22.04.04 from snap (not manually).
I want to put some good settings and choose Alsa output for example.

However I just have Pulseaudio on output.

Why?

Thanks in advance for your answers

**Comment by VincentLeclercq** [comment]
I precise that I didn't find a graphical way to install plugins, and the installation of plugins didn't work in the terminal.

configure: error: in `/home/libres/Téléchargements/audacious-plugins-4.3.1':
configure: error: The pkg-config script could not be found or is too old.  Make sure it
is in your PATH or set the PKG_CONFIG environment variable to the full
path to pkg-config.

Alternatively, you may set the environment variables GLIB_CFLAGS
and GLIB_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

To get pkg-config, see <http://pkg-config.freedesktop.org/>.
See `config.log' for more details

**Comment by radioactiveman** [comment]
We are not in charge for the Snap package. Please ask the responsible publisher why they include only the PulseAudio output plugin.

For Ubuntu we recommend to install the latest Audacious version with a PPA, look into the Ubuntu chapter in our [download instructions](https://audacious-media-player.org/download).
